contrib/charts/cert-manager/templates/deployment.yaml

```diff
@@ -44,7 +44,15 @@ spec:
           resources:
 {{ toYaml .Values.ingressShim.resources | indent 12 }}
 {{- end }}
-{{- if .Values.nodeSelector }}
+    {{- with .Values.nodeSelector }}
```

Comment:
Not sure why the change to `with` from `if`? Could we keep this (and the new affinity and tolerations stanzas) using `if` here? Else our generated deployment manifests will end out with blank `nodeSelector`, `affinity` and `tolerations` fields.

Reply:
@munnerz the `with` statement handles this internally - if the value is not present or empty - the contents of with `with` block are skipped. This is the pattern you'll now see `helm create foo`  generating for you from the standard chart template.